src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -1154,7 +1154,7 @@ def __init__(self, password_arg_value, use_cert_sn_issuer=None):
                         match = re.search(r'\-+BEGIN CERTIFICATE.+\-+(?P<public>[^-]+)\-+END CERTIFICATE.+\-+',
                                           self.cert_file_string, re.I)
                         self.public_certificate = match.group('public').strip()
-            except UnicodeDecodeError:
+            except [UnicodeDecodeError, Error]:
```

Comment:
Better to use a `tuple`, instead of a `list` to capture exceptions. See [Handling Exceptions](https://docs.python.org/3/tutorial/errors.html#handling-exceptions).